### PR TITLE
content/quickstart: Add Node tracing quickstart

### PR DESCRIPTION
**Pull request by hvent90**
[Image](https://user-images.githubusercontent.com/5974764/44645101-47813b00-a98b-11e8-8e31-ee48b20b3c61.png) (image capturing software is slightly buggy)

@eduardoemery, sorry if I am stepping on your toes by submitting this PR. I was eager to dive in to the Node implementation of OpenCensus as Javascript is my most fluent language.

### CODE REVIEW

**Comment by eduardoemery** [comment]
This is really great! I'll take a look as soon as possible and see if I can help with anything :smile:

File: content/quickstart/node/tracing.md [hunk 1]
```diff
@@ -0,0 +1,697 @@
...
+Put this in your newly generated `index.js` file:
+
+```js
+const stdin = process.openStdin();
+
+function readEvaluateProcessLine(input) {
+  const text = readLine(input);
+  const upperCase = processLine(text);
+
+  console.log(upperCase);
```
Comment (odeke-em) [inline content/quickstart/node/tracing.md]: So the intro shows
```shell
> foo
< FOO
```
but the code would plainly print
```shell
foo
FOO
```
Perhaps please update the intro or better let's print without newlines using `process.stdout.write` e.g. for the prompt
```javascript
process.stdout.write('> ');
const text = readLine(input);
const upperCase = processLine(text);
process.stdout.write('\n< ' + upperCase + '\n\n');
```

**Comment by odeke-em** [comment]
>This is really great! I'll take a look as soon as possible and see if I can help with anything 😄

Thank you @eduardoemery! Actually could we use @hvent90's PR here as the part for tracing because it provides screenshots, too and was earlier and then use yours for metrics, that way both of your work is in :)

**Comment by odeke-em** [comment]
@hvent90 btw your last screenshot exposes private information about your Stackdriver/GCP project `organization-id`, please recreate it.

File: content/quickstart/node/_index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: "Node"
```
[the remark this answers is not in the document]
Comment (hvent90) [reply content/quickstart/node/_index.md]: I don't understand this, can you rephrase?

File: content/quickstart/node/tracing.md (hunk 1 above)
Comment (hvent90) [reply content/quickstart/node/tracing.md]: Thank you for the help, CLI wizard!

**Comment by hvent90** [comment]
Pushing up a commit with these changes:

1. Removed any mention of `>` and `<`
2. Removed my project ID from last code snippet
3. Added new photos that exclude header and nav bar of Stackdriver

**Comment by eduardoemery** [comment]
Hey @odeke-em I love your idea. Henry has great screenshots on his quickstart and, as I said, both his and mine end up doing the same thing. I'll update #289 to only include the metrics quickstart :smile:

**Comment by hvent90** [comment]
@eduardoemery sorry again for submitting this while you were working on it. I see you've authored a ton of the code over at the `opencensus-node` repo, and I want to thank you for providing such an awesome library!

**Comment by eduardoemery** [comment]
Don't worry! Actually, this makes me very happy. This means that the entire team could made a great implementation that allowed other developers to use, understand and contribute to it!

File: content/quickstart/node/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (odeke-em) [reply content/quickstart/node/_index.md]: Please replace the title "Node" with "Node.js", the term "Node.js" is more recognizable and the trademark of the language.

File: content/quickstart/node/tracing.md [hunk 3]
```diff
@@ -75,8 +75,8 @@ stdin.addListener("data", readEvaluateProcessLine);
 ## Getting Started
 The Repl application takes input from users, converts any lower-case letters into upper-case letters, and echoes the result back to the user, for example:
 ```bash
-> foo
```
Comment (odeke-em) [inline content/quickstart/node/tracing.md]: Would you mind actually using the suggestion for `process.stdout.write`? That way we can achieve the consistency that we'd want with other languages.

File: content/quickstart/node/tracing.md [hunk 4]
```diff
@@ -633,7 +633,7 @@ const defaultConfig = {
   samplingRate: 1.0  // always sample
 };
 
-const exporter = new stackdriver.StackdriverTraceExporter({projectId: 'test-210722'});
+const exporter = new stackdriver.StackdriverTraceExporter({projectId: 'your-project-id'});
```
Comment (odeke-em) [inline content/quickstart/node/tracing.md]: In other languages we've used `GCP_PROJECTID` instead of a string.

In Node.js, you can retrieve from the environment using `process.env.GCP_PROJECTID`, please use that.

Comment (hvent90) [reply content/quickstart/node/tracing.md]: `process.env.GCP_PROJECTID` returned null, this is how I ended up retrieving the id:

```js
function getProjectId() {
  return require(process.env.GOOGLE_APPLICATION_CREDENTIALS).project_id;
}

const exporter = new stackdriver.StackdriverTraceExporter({projectId: getProjectId()});
```
Let me know if my implementation isn't what you meant! Thanks again for reviewing.

Comment (odeke-em) [reply content/quickstart/node/tracing.md]: Gotcha, thanks! That can work but in Javascript you can also do `const projectId = process.env.GCP_PROJECTID || 'census-demos'` as the other demos do. However, let's merge this in, yours is perfect as is, thank you!
